ci/dev-pipeline.yml

```diff
@@ -67,7 +70,7 @@ resources:
   - name: tag_commit
     type: git
     icon: github
-    check_every: never
+    check_every: 1m
```

Comment:
This is to check for changes on the pipeline so concourse runs. @JasonBellONS  helped me to get the pipeline going when it was not running.

@JasonBellONS am I right here?

Reply:
Yep, this was so James could test his changes on a dev environment, once this pr is otherwise ready we can change this back to never and set the live-pipeline.yml here as as live-pipeline